### PR TITLE
feat(trends): Rename trendsUnselectedSeries query param to unselectedSeries for consistency

### PR DESCRIPTION
**Pull request by udameli**
Track unselected legend items as `unselectedSeries` instead of trends-specific `trendsUnselectedSeries` so that selection is persisted across all display views in transaction summary view

### CODE REVIEW

File: static/app/components/charts/utils.tsx [hunk 1]
```diff
@@ -185,10 +185,9 @@ export function shouldFetchPreviousPeriod({
  * Generates a series selection based on the query parameters defined by the location.
  */
 export function getSeriesSelection(
-  location: Location,
-  parameter = 'unselectedSeries'
```
Comment (udameli) [inline static/app/components/charts/utils.tsx:189]: since it's always going to be `unselectedSeries`, the parameter is no longer needed to be dynamic

Comment (k-fish) [inline static/app/components/charts/utils.tsx:189]: Nice cleanup!

**Review by k-fish — APPROVED** [review]
🚢

File: static/app/components/charts/utils.tsx [hunk 2]
```diff
@@ -185,10 +185,9 @@ export function shouldFetchPreviousPeriod({
...
+  location: Location
 ): LegendComponentOption['selected'] {
-  const unselectedSeries = decodeList(location?.query[parameter]);
+  const unselectedSeries = decodeList(location?.query.unselectedSeries);
```
Comment (Zylphrex) [inline static/app/components/charts/utils.tsx:190]: Nit: assuming `location` is a required field, the `?` isn't necessary here.

```suggestion
  const unselectedSeries = decodeList(location.query.unselectedSeries);
```

**Review by Zylphrex — APPROVED** [review]

File: static/app/components/charts/utils.tsx (hunk 2 above)
Comment (udameli) [reply static/app/components/charts/utils.tsx:190]: done!

**Pull request merged**